examples/lighting-app/efr32/BUILD.gn

```diff
@@ -60,6 +60,9 @@ declare_args() {
 
   # Disable LCD on supported devices
   disable_lcd = false
+
+  # Disable IPv4 for rs911
+  chip_disable_wifi_ipv4 = false
```

Comment:
This line will enable IPv4, not disable it.

Reply:
@sharad-patil24 is this the desired default behaviour ?

Reply:
By default IPv4 and IPv6 is enabled. IPv4 can be disable during the build i.e., changing this value to true. Maybe the comment "Disable IPv4 for rs911"  should be changed